### PR TITLE
Fixes #25.

### LINKED ISSUE #25: Import production database to local development database

### PR DESCRIPTION
**Pull request by jweakley**

### CODE REVIEW

**Comment by chadzink** [comment]
Seems good, and cool. Maybe have Sam take a look too?

**Comment by jweakley** [comment]
@Samsinite can you take a look at this when you get a chance?

File: lib/wildland_dev_tools/heroku.rb [hunk 1]
```diff
@@ -50,6 +50,16 @@ def deploy_master_to_staging(verbose = false)
...
 
+      def import_production_database(verbose = false)
+        puts 'Determining heroku app names.' if verbose
+        production_app_name = get_app_name('production', verbose)
+        scratch_file_name = 'latest.dump'
+        database_name = Rails.configuration.database_configuration['development']['database']
+        download_database(production_app_name, scratch_file_name, verbose)
+        import_database(database_name, scratch_file_name, verbose)
+        File.delete(scratch_file_name)
+      end
```
Comment (Samsinite) [inline lib/wildland_dev_tools/heroku.rb:61]: Passing around the state here screams the need for encapsulation into a class and not a module. For example, verbose, app name, database name, etc...

**Comment by Samsinite** [comment]
I'm fine with merging what is here, but at some point it could use a bit of cleanup into some classes to simplify updates. I suggest the next time this library is touched, that the parts of the modules being touched be pulled out into some classes

**Pull request merged**